lib/build-imports-map.js

```diff
@@ -0,0 +1,76 @@
+/* jshint node:true */
+'use strict';
+
+// TODO have this build the map based on the scss import
+// dependency map instead of the styles directory structure
```

Comment:
it's just based off the components directory right now, should only need to change what this file does to make it based on the scss import dependency tree